ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/Record_Decision/103195_create-approved-decision.js

```diff
@@ -50,11 +49,12 @@ describe('103195 Record new Approved decision', () => {
         cy.projectStateId().should('contain.text', 'APPROVED')
     })
 
+    // Displayed at the end of the journey
     function checkSummary() {
         cy.ApprovedDecisionPreview().should('contain.text', 'APPROVED WITH CONDITIONS')
         cy.ApprovedMadeByPreview().should('contain.text', 'Regional Director for the region')
         cy.AprrovedConditionsSet().should('contain.text', 'Yes')
-        cy.ApprovedConditionDetails().should('contain.text', 'This is a test')
+        cy.AprrovedConditionsSet().should('contain.text', 'This is a test')
```

Comment:
This looks like a typo, should be - ApprovedConditionsSet

Reply:
Perhaps the command itself is mispelt.

Reply:
Yes; I'm a bit hesitant to "fix" typos after the last one affected so much. Maybe on a separate task/PR?